test/e2e/testdata/wolf-rbac/db-psql.sql

```diff
@@ -0,0 +1,223 @@
+
+-- you can change the password on initial the database.
+/**
+CREATE USER wolfroot WITH PASSWORD '123456';
+CREATE DATABASE wolf with owner=wolfroot ENCODING='UTF8';
+GRANT ALL PRIVILEGES ON DATABASE wolf to wolfroot;
+*/
+
+\c wolf;
```

Comment:
As an extra, was this file written by you, or copied the content of other projects? I just want to confirm the correctness of the license

Reply:
This is the content of `wolf-rbac` projects. It is used to initialize the database. The `wolf-server` cannot work normally without this.